velox/expression/tests/ArgumentTypeFuzzer.cpp

```diff
@@ -48,7 +48,24 @@ void ArgumentTypeFuzzer::determineUnboundedTypeVariables() {
     // Random randomType() never generates unknown here.
     // TODO: we should extend randomType types and exclude unknown based
     // on variableInfo.
-    bindings_[variableName] = randType();
+    // Respect orderable/comparable constraint.
+    TypePtr typePtr = nullptr;
```

Comment:
Let's move this to a separate PR and add a test. See velox/expression/tests/ArgumentTypeFuzzerTest.cpp